### ISSUE TITLE
How for Forward Audio (Input & Output and every Audio) from Phone to PC (Ubuntu)

### ISSUE THREAD

**Issue by luisalvarado** [Genymobile/scrcpy#5314]
I read the following part in the documentation:

--audio-source=source
        Select the audio source (output, mic or playback).
        The "output" source forwards the whole audio output, and disables playback on the device.
        The "playback" source captures the audio playback (Android apps can opt-out, so the whole output is not necessarily captured).
        The "mic" source captures the microphone.
        Default is output.

But can I forward all of them, meaning, 

--audio-source=output,playback,mic

something like this so that I can:

1. Hear the game sound on PC
2. Receive call and respond on PC (Hear the phone audio and me talk on the PC and the microphone audio goes to the phone
3. Play audio of apps.

I can see some work sometimes, other never worked since I started testing in 2.3.x For example I have not received a call on the phone and was able to actually talk on the computer (Ubuntu 24.04 now).

The phone is an Samsung 21 Plus, a 22 Ultra and a 24 Ultra. All have Android 14 and OneUI 6.1 (In case the OneUI helps).

I read that it was supposed to be native audio forwarding in 12, but I am in 14 and for example, when I play Stardew, yes, the sound is automatically forwarded to the Computer (Which on a USB 3.2 it still has FPS stuttering  on both, Cable USB and even testing on Wifi on a Wifi 6 router). 

But when I receive a call, a normal phone call, I can not hear it on the computer, nor can I respond via the PC.

I am using 2.6.1 right now and I only run the command scrcpy and it connects via USB without a single issue, but still can not take phone calls on the PC.

So to summarize:

Using Ubuntu 24.04, with scrcpy 2.6.1, everything is beautiful except when receiving a phone call. Telegram works perfectly, audio for games works perfectly, even youtube etc works. Audio works for all of them dandy.

But a phone call does not work. What special parameter should I be using for this in case I am missing something here. Thank you and big hugs to the team for making a miracle app.

**Comment by rom1v** [comment]
Phone calls cannot be captures/forwarded. PC microphone cannot be injected to Android.

**Comment by luisalvarado** [comment]
Thank you @rom1v  I am guessing for security reasons it is not possible.
